examples/tavily-agent/.env.example

```
@@ -0,0 +1,2 @@
+TAVILY_API_KEY=
+ANTHROPIC_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ANTHROPIC_API_KEY key should go before the TAVILY_API_KEY key

(UnorderedKey)